src/test/java/org/cactoos/func/IoCheckedFuncTest.java

```diff
@@ -47,9 +47,11 @@ public void rethrowsIoException() {
                 }
             ).apply(1);
         } catch (final IOException ex) {
-            MatcherAssert.assertThat(
-                ex, Matchers.is(exception)
-            );
+            new Assertion<>(
```

Comment:
@andreoss all of those should be transformed to use `Assertion` with `Throws`

Reply:
@victornoel Done